src/pages/menu-page/MenuPage.jsx

```diff
@@ -1,79 +1,136 @@
+        }));
+    }
+
+    handleMouseEnter = () => {
+        this.setState({ isHovered: true });
+    };
+
+    handleMouseLeave = () => {
+        this.setState({ isHovered: false });
+    };
+
+    handleButtonClick = (buttonName) => {
+        if (this.state.activeButton !== buttonName) {
+            this.setState({ activeButton: buttonName });
         }
     };
 
-    return (
-        <>
-            <main>
-                <div className='mainContainer'>
-                    <div className='browseInfoContainer'>
-                        <div className='browseMenuTitle'>
-                            Browse our menu
-                        </div>
-                        <div className='browseMenuDescription'>
-                            Use our menu to place an order online, or
-                            {''}
-                            <div className='toolTipWrapper'
-                                 onMouseEnter={() => setIsHovered(true)}
-                                 onMouseLeave={() => setIsHovered(false)}>
-                                <span className='phone-ToolTip'>
-                                &nbsp;phone&nbsp;
-                                    {isHovered && (
-                                        <div><PhoneNumberToolTip/></div>
-                                    )}
-                            </span>
-                            </div>
-                            {''}
-                            our store to place a pickup order. Fast and
-                            fresh food.
-                        </div>
-                        <div className='browseMenuButtons'>
-                            <div>
-                                <button
-                                    onClick={() => handleButtonClick('Desert')}
-                                    className={activeButton === 'Desert' ? 'active' : ''}>
-                                    Desert
-                                </button>
+    render() {
+        const { isHovered, activeButton, meals, loading, error } = this.state;
+
+        return (
+            <>
+                <main>
+                    <div className='mainContainer'>
+                        <div className='browseInfoContainer'>
+                            <div className='browseMenuTitle'>
+                                Browse our menu
                             </div>
-                            <div>
-                                <button
-                                    onClick={() => handleButtonClick('Dinner')}
-                                    className={activeButton === 'Dinner' ? 'active' : ''}>
-                                    Dinner
-                                </button>
+                            <div className='browseMenuDescription'>
+                                Use our menu to place an order online, or
+                                {''}
+                                <div
```

Comment:
The behavior should be that you wrap some text/component with <Tooltip>Phone number </Tooltip> and it should work, without additional wrappers at the place of usage